test/util/calculation_type.cpp

```diff
@@ -9,12 +9,19 @@
 // Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 // http://www.boost.org/LICENSE_1_0.txt)
 
+#if !defined(BOOST_TEST_MODULE)
+# define BOOST_TEST_MODULE Boost.Geometry Util calculation_type
+#endif
 
 #include <geometry_test_common.hpp>
 
 #include <boost/geometry/geometries/geometries.hpp>
 #include <boost/geometry/util/calculation_type.hpp>
 
+using namespace boost::unit_test;
```

Comment:
That's my mistake, this is used only when manually registering test suites / cases. It should be removed here.